packages/passport-interface/src/RequestTypes.ts

```diff
@@ -357,10 +351,10 @@ export type VerifyTokenRequest = {
 };
 
 /**
- * On the happy path, we don't need to say anything in response to
+ * Returns the encryption_key of the account, if stored on our server.
```

Comment:
probably worth clarifying that this encryption key is only stored on our server in the case that a user has opted to not set a password to avoid scaring the folks